### PR TITLE
Private key import

### PR DESCRIPTION
**Pull request by calvinmetcalf**
node 5.2.0 added an improvement to make importing private keys easier. This also updates the tests to test for it and tests against node when it's available.

### CODE REVIEW

**Comment by calvinmetcalf** [comment]
this is green but there looks to be an intermittent error showing up, need to test if it's related to this pull.

File: test.js [hunk 1]
```diff
@@ -59,6 +59,24 @@ function run(i) {
 			var pub2 = dh2.computeSecret(pubk1).toString('hex');
 			t.equals(pub1, pub2, 'equal secrets');
 		});
+    test(mod + ' run ' + i + ' new way to set stuff', function (t){
```
Comment (dcousens) [inline test.js]: whats going on with the indentation here?

File: test.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 var test = require('tape');
-var nodeCrypto  = require('./browser');
+var nodeCrypto  = require('./');
```
Comment (dcousens) [inline test.js]: shouldn't we be testing `./browser`?

Comment (calvinmetcalf) [inline test.js]: No I want to test node against our code but I want it to also work in older node which doesn't support ecdsa

File: test.js (hunk 1 above)
Comment (calvinmetcalf) [inline test.js]: Probably my bad I can fix

**Comment by calvinmetcalf** [comment]
woops looks like if you get the public key and then set the private key the public key is not regenerated

**Pull request merged**